### ISSUE TITLE
[REQUEST] Urban Trial Freestyle

### ISSUE THREAD

**Issue by ChiSys** [Electry/VitaGrafix#50]
It seems to run lower than with native res. Would be great if you could look into it.

**Comment by Electry** [comment]
Hey, just wanted to let you know that I've managed to get it to run at 960x544, but the performance is really bad.

Menu is running at around 15 fps and during gameplay it's anywhere between 20-30 fps.
This is at max clocks, 494/222.

![2018-12-26-005928](https://user-images.githubusercontent.com/12598379/50427490-13227a80-08aa-11e9-93f4-d444995c6304.png)

![2018-12-26-010100](https://user-images.githubusercontent.com/12598379/50427493-1ae21f00-08aa-11e9-943a-129d90122ea5.png)

I couldn't get it to run when I tried before, and the reason was that they hardcoded memory block sizes in the eboot *facepalm*, so if you raise the RT size to 960x544 it just wouldn't fit in 2 MBs of allocated VRAM memory (or whatever was the exact figure). And also, Depth-Stencil stride sample size has to be a multiple of 32, otherwise you get pink screen and nothing more (it's 736 by default).

I can still include it if you want, but it's not that great experience.
